controller/decorator/controller.go

```diff
@@ -585,7 +598,8 @@ func (c *decoratorController) getChildren(parent *unstructured.Unstructured) (co
 	for _, child := range c.dc.Spec.Attachments {
 		// List all objects of the child kind in the parent object's namespace,
 		// or in all namespaces if the parent is cluster-scoped.
-		informer := c.childInformers.Get(child.APIVersion, child.Resource)
+		groupVersion, _ := schema.ParseGroupVersion(child.APIVersion)
```

Comment:
Are we ok to ignore err (i.e. second response)?

Reply:
I have added validation in method `newDecoratorController` in lines 154+, at this point all those apigroups are validated (as, in case of errors, it would be thrown there)